commit 6a677b050c91000d7a196dd764a9aca5dda0f98f
Author: q66 <q66@chimera-linux.org>
Date:   Fri Sep 23 02:43:48 2022 +0200

    disable probably broken function that is only built with clang
    
    The __ppc macro is never defined on ppc gcc.

diff --git a/sqlite3.c b/sqlite3.c
index 946815f..c70fcfc 100644
--- a/sqlite3.c
+++ b/sqlite3.c
@@ -36855,7 +36855,7 @@ SQLITE_PRIVATE int sqlite3VListNameToNum(VList *pIn, const char *zName, int nNam
      return (sqlite_uint64)hi << 32 | lo;
   }
 
-#elif !defined(__STRICT_ANSI__) && (defined(__GNUC__) && defined(__ppc__))
+#elif 0
 
   __inline__ sqlite_uint64 sqlite3Hwtime(void){
       unsigned long long retval;
